### DISCUSSION TITLE
Live streaming

### DISCUSSION THREAD

**Discussion by christianfunginet** [flutterjanus/flutter_janus_client#162] [Q&A]
Hi everyone. I need help to make live video streaming in flutter. Someone have achieved something like that? Thaks a lot.

**Comment by shivanshtalwar** [comment]
Well to achieve that you have to build janus driving logic  by yourself, to put it simply
You use videoroom plugin join as a publisher and then rtp forward your stream to streaming plugin then once you have context of both room and streaming mount point using janus_client attach both the plugins and setup webrtc negotiations and that's it
If you are interested i am also building a startup called januscaler which is janus on steroids in terms of scalability and features, you can look at the landing page at 
https://januscaler.com

**Reply by christianfunginet** [reply]
Thanks a lot. I Will take a look at that repo.
